tests/gis_tests/gdal_tests/test_raster.py

```diff
@@ -103,7 +103,17 @@ def test_geotransform_and_friends(self):
         self.assertEqual(self.rs.skew.y, 0)
         # Create in-memory rasters and change gtvalues
         rsmem = GDALRaster(JSON_RASTER)
+        rsmem.geotransform = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
```

Comment:
This looks redundant with the next line. I guess it's for additional test coverage (which is typically better in a separate commit than along side a bug fix) but that might not be clear when reading this code in the future. Perhaps a comment should clarify.